ui/pages/confirmations/confirm-token-transaction-base/confirm-token-transaction-base.js

```diff
@@ -59,7 +59,7 @@ export default function ConfirmTokenTransactionBase({
 
   const ethTransactionTotalMaxAmount = Number(
     hexWEIToDecETH(hexMaximumTransactionFee),
-  );
+  ).toFixed(8);
```

Comment:
I think `8` is already default value used for formatting, unless overridden https://github.com/MetaMask/metamask-extension/blob/74678b0080602a578e820befbcb15aba0dbcc1ce/ui/hooks/useUserPreferencedCurrency.js#L58

Reply:
interesting. we do have existing code that may show more decimals. Not finding specific reasoning as to why these numbers were chosen. That said, since this value is specifically for ETH tokens, 8 seems fine to me as it's the fallback you mentioned here and our fiat decimals are limited. Thanks for pointing this out! 

<img width="809" alt="Screenshot 2024-04-30 at 10 51 59 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/b3c123d5-bbc7-403c-9308-acccc16872d5">